moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -134,10 +134,8 @@ class Servo
 private:
   /**
    * \brief Convert a give twist command to planning frame
-   * This implementation does not differentiate between body twist and spatial twist,
-   * i.e if the command is as seen from a stationary or non-stationary frame.
-   * The result of this transformation will only be accurate if the command frame is a stationary frame.
-   * See issue: https://github.com/ros-planning/moveit2/issues/2150
+   * The reference frame specified by `frame_id` should be a stationary frame.
+   * The command maybe expressed in the space frame basis or the body frame basis.
```

Comment:
```suggestion
   * The command may be expressed in the space frame basis or the body frame basis.
```

I also think more explanation here would be required. What does space frame vs. body frame mean?